eth/rpc/backend/backend_suite_test.go

```
@@ -167,35 +171,41 @@
+			return nil, nil, nil
 		default:
 			time.Sleep(1 * time.Second)
 		}
 	}
 }
 
-// getCurrentNonce returns the current nonce of the funded account
-func (s *BackendSuite) getCurrentNonce(address gethcommon.Address) uint64 {
-	bn, err := s.backend.BlockNumber()
+// getUnibiBalance returns the balance of an address in unibi
+func (s *BackendSuite) getUnibiBalance(address gethcommon.Address) *big.Int {
```

golangci-lint (1.62.2)

[error] func `(*BackendSuite).getUnibiBalance` is unused

(unused)